tests/phpunit/CRM/Core/ErrorTest.php

```diff
@@ -106,8 +106,7 @@ public function testDebugLoggerFormat(): void {
     $log->log('Mary had a little lamb');
     $log->log('Little lamb');
     $config = CRM_Core_Config::singleton();
-    $fileContents = file_get_contents($log->_filename);
-    $this->assertEquals($config->configAndLogDir . 'CiviCRM.' . CIVICRM_DOMAIN_ID . '.' . 'my-test.' . CRM_Core_Error::generateLogFileHash($config) . '.log', $log->_filename);
+    $fileContents = file_get_contents($config->configAndLogDir . 'CiviCRM.' . CIVICRM_DOMAIN_ID . '.' . 'my-test.' . CRM_Core_Error::generateLogFileHash($config) . '.log');
```

Comment:
I can see they've prevented getting the log file name now, just it means two things:
* The update to 1.14 will break the [logviewer extension](https://github.com/adixon/ca.civicrm.logviewer/blob/a937114b96b05c0e3efcbec31cf662df63d79612/CRM/Logviewer/Page/LogViewer.php#L8). FYI @adixon 
* While maybe not the most important thing in the world, the naming of the file was a relevant thing to test. And especially now if we have to know how it's constructed in order to find it, then that structure should be enforced by a test.

Does our function CRM_Core_Error::generateLogFileName() need to be protected? Could we expose that? The hash function is already public. The Civi::static is public but people shouldn't rely on that.